common/djangoapps/student/cookies.py

```diff
@@ -8,16 +8,23 @@
 
 import six
 from django.conf import settings
-from django.contrib.auth.models import User
+from django.contrib.auth.models import User, AnonymousUser
```

Comment:
Not a blocker, but wondering when we can start moving code out of the student djangoapp.